client/Overview/FeaturedReviewLink.jsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
-// import Stars from 'client/SharedComponents/Stars.jsx';
+// import Stars from '../SharedComponents/Stars.jsx';
 
 // Kyle: I did also create a Stars component that may be of use to you. It's in the SharedComponents folder. All you have to pass it is a rating number as a prop, and it will give you an appropriate amount of filled stars.
 
+// function ScrollToReviewSection() {
+//   document.getElementByClassName('ratings-and-reviews-container').scrollIntoView(true)
+// }
+
 function FeaturedReviewLink() {
 // const FeaturedReviewLink = () => {
 // error: Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style
   return (
     <div id="productAreaStars">
-      <h5>Placeholder: [*****] Read all reviews</h5>
+      <strong>[*****]</strong>
```

Comment:
You should be able to replace this with a <Stars rating={5} /> component here. Or did that not work with your CSS somehow? I know Sam was having some trouble positioning it correctly.